config_parser_test.cc

```diff
@@ -1,11 +1,30 @@
 #include "gtest/gtest.h"
 #include "config_parser.h"
 
-TEST(NginxConfigParserTest, SimpleConfig) {
-  NginxConfigParser parser;
-  NginxConfig out_config;
+class NginxConfigParserTest : public ::testing::Test{
+	protected:
+		NginxConfigParser parser;
+  		NginxConfig out_config;
+};
+
+TEST_F(NginxConfigParserTest, SimpleConfig) {
 
   bool success = parser.Parse("example_config", &out_config);
 
   EXPECT_TRUE(success);
+  EXPECT_EQ(out_config.ToString(0), "foo \"bar\";\n");
+
+  bool success2 = parser.Parse("example_config2", &out_config);
+  EXPECT_FALSE(success2);
+}
+
+TEST_F(NginxConfigParserTest, TestInput){
+	bool success = parser.Parse("invalideof", &out_config);
```

Comment:
To make the code a bit easier to read, you could include a "ParseString" method in your test fixture, that way you don't have to keep writing that second argument of the "parser.Parse" method